library/item/bows/src/testmod/java/org/quiltmc/qsl/bows/BowsTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.bows;
+
+import org.jetbrains.annotations.NotNull;
+import org.quiltmc.qsl.bows.api.ExtendedBowItem;
+import org.quiltmc.qsl.bows.api.ShotProjectileEvents;
+import org.quiltmc.qsl.bows.api.ExtendedCrossbowItem;
+
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.projectile.PersistentProjectileEntity;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemGroup;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+
+import net.fabricmc.api.ModInitializer;
+
+public class BowsTest implements ModInitializer {
```

Comment:
I'm surprised this passed the test in 1.18.2, but it still should be changed to QSL's ModIntializer.